src/Microsoft.IdentityModel.Tokens/SecurityAlgorithms.cs

```diff
@@ -113,6 +113,9 @@ public static class SecurityAlgorithms
         public const string Aes192CbcHmacSha384 = "A192CBC-HS384";
         public const string Aes256CbcHmacSha512 = "A256CBC-HS512";
 
+        internal const string DefaultAsymmetricKeyWrapAlgorithm = RsaOaepKeyWrap;
+        internal const string DefaultSymmetricAlgorithm = Aes128CbcHmacSha256;
```

Comment:
suggest:
DefaultSymmetricAlgorithm -> DefaultSymmetricEncryptionAlgorithm

there are other SymmetricAlgorithms such as Signing.

Reply:
it makes sense - I'll rename.